scripts/hdl/simify_verilog.py

```diff
@@ -79,6 +101,9 @@ def add_vcd(contents: str) -> str:
 
     return contents.replace(orig_str, f"{orig_str} {vcd_blob}")
 
+'''
+args = ['scripts/hdl/simify_verilog.py', 'build/Core.raw.v']
+'''
```

Comment:
Spurious?

Reply:
Sorry, I think this is my mistake. I will delete this.